aodndata/aatams/aatams_sattag_qc_dm_schema.py

```diff
@@ -281,21 +287,21 @@ def is_equal(x, y):
 CSV_LATITUDE = And(CSV_FLOAT, Or(valid_latitude, 0))  # as above
 
 METADATA_SCHEMA = {
-    "sattag_program": str,  # TODO
+    "sattag_program": str,
     "device_id": valid_device,
     "ptt": CSV_INT,
-    "body": str,  # TODO
+    "body": str,
     "device_wmo_ref": valid_wmo_device,
-    "tag_type": valid_tag_type,  # TODO
-    "common_name": str,  # TODO
-    "species": str,  # TODO
+    "tag_type": valid_tag_type,
+    "common_name": str,
+    "species": str,
     "release_longitude": CSV_LONGITUDE,
     "release_latitude": CSV_LATITUDE,
-    "release_site": str,  # TODO
+    "release_site": str,
     "release_date": Or(CSV_EMPTY, CSV_DATE_ISO, CSV_DATE_US),
     "recovery_date": Or(CSV_EMPTY, CSV_DATE_ISO, CSV_DATE_US),
-    "age_class": CSV_AGE_CLASS,  # TODO not applicable is case!?
-    "sex": CSV_SEX_CLASS,  # TODO not applicable is case!?
+    "age_class": CSV_AGE_CLASS,
```

Comment:
just a check that's a string is fine (see also my comment above on `CSV_SEX_CLASS`)

Reply:
done